python/cuml/solvers/qn.pyx

```diff
@@ -208,6 +210,10 @@ class QN(Base,
         Maximum number of iterations taken for the solvers to converge.
     tol: float (default = 1e-3)
         The training process will stop if current_loss > previous_loss - tol
+    delta: Optional[float] (default = None)
+        The training process will stop if
```

Comment:
Since this is a parameter that doesn't exist in sklearn, it might be good to add a brief intro or one line explanation of why it exists and how it can affect behavior (which will be easier for many people as opposed to having to analyze the implications of the mathematical explanation below).